cobigen-templates/templates-oasp4j/src/main/templates/crud_java_server_app/templates.xml

```diff
@@ -3,8 +3,8 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.1">
 
   <templates>
-    <templateExtension ref="${variables.component#cap_first}.java" mergeStrategy="javamerge"/>
-    <templateExtension ref="${variables.component#cap_first}Impl.java" mergeStrategy="javamerge"/>    
+    <!-- <templateExtension ref="${variables.component#cap_first}.java" mergeStrategy="javamerge"/> -->
+    <!-- <templateExtension ref="${variables.component#cap_first}Impl.java" mergeStrategy="javamerge"/> -->    
```

Comment:
Component interface is not mergeable anymore? That's strange. It would disable generation iteratively for multiple entities in the same component.